src/constants.scss

```diff
@@ -3,3 +3,14 @@ $background-light-accent: #bbbbbb;
 $background-dark-accent: #999999;
 
 $text-colour: #000000;
+
+$dark-blue: #28536b;
+$grey-blue: #8ba6a9;
+$light-blue: #a7cecb;
+$dark-orange: #cf5c36;
+$light-orange: #f1a66a;
+
+$primary: $dark-orange;
+$secondary: $dark-blue;
+$primary-light: $light-orange;
+$secondary-light: $light-blue;
```

Comment:
Weren't these added in a different PR? If so, I imagine these changes will go away once the other is merged into `staging` 😊

Reply:
ah, i thought so, it think we just replicated it so we could see for the page design